tools/src/faircv/Main.hs

```diff
@@ -0,0 +1,107 @@
+
+import Data.Aeson
+import qualified Data.Set as Set
+import Data.Traversable
+import Data.Time.Clock
+import Servant.Client
+import Test.QuickCheck
+
+import Loot.Log
+
+import Dscp.Core
+import Dscp.Crypto
+import Dscp.Witness.Web hiding (checkFairCV)
+import Dscp.Resource.Keys
+
+import Client
+
+createActor :: IO (SecretKey, PublicKey, Address)
+createActor = do
+    sk   <- genSecretKey
+    pk   <- return (toPublic sk)
+    addr <- return (mkAddr   pk)
+    return (sk, pk, addr)
+
+createActorOf :: SecretKey -> IO (PublicKey, Address)
+createActorOf sk = do
+    pk   <- return (toPublic sk)
+    addr <- return (mkAddr   pk)
+    return (pk, addr)
+
+instance MonadLogging IO where
+    log = putStrLn . fmtMessageColored
+
+    logName = return CallstackName
+
+main :: IO ()
+main = do
+    [url, skFile] <- getArgs
+
+    wClient <- createWitnessClient =<< parseBaseUrl url
+
+    store <- readStore skFile emptyPassPhrase
```

Comment:
As I mentioned in slack, if you submit a publication transaction in place of real educator, then his private chain gets broken. Maybe it would be easier to generate a random key (or key from seed).